modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -42,6 +43,35 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.13.1
+
+Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.13.0 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
+
+=== Open in Web Improvements ===
+
+The "Open in Web" feature for direct editing of office documents now properly works on iOS and Desktop clients with Collabora. 
+
+=== Avoid Loading 3d-party Resources
+
+The "Open in Web" configuration offers icons to be shown in the clients. This is e.g. used by the iOS 12.0.3 client. We now ship the icons embedded in core.
+
+=== Notable Changes
+
+* Fix: disallow pre-signed url access if the signing key is not initialized.
+* New code was added to dismiss invalid settings of the redirection endpoint URI as seen in the OAuth2 protocol, according to RFC#7636.
+
+=== Updated App Versions
+
+Find below a list of updated apps in comparison with the 10.12.2 complete bundle. More information on the changes can be found in the respective changelogs on ownCloud Marketplace.
+
+* https://marketplace.owncloud.com/apps/graphapi [Graph API]: v0.3.1
+* https://marketplace.owncloud.com/apps/guests [Guests] 0.12.3
```

Comment:
Is the change for whitelisted apps going to be in guests 0.12.3 and bundled with 10.13.1 ?

Reply:
yes.